subprojects/docs/src/samples/userguide/dependencyManagement/workingWithDependencies/accessMetadataArtifact/groovy/build.gradle

```diff
@@ -22,9 +24,9 @@ task printGuavaMetadata {
             Set<ArtifactResult> mavenPomArtifacts = component.getArtifacts(MavenPomArtifact)
             ArtifactResult guavaPomArtifact = mavenPomArtifacts.find { it.file.name == 'guava-18.0.pom' }
             def xml = new XmlSlurper().parse(guavaPomArtifact.file)
-            println guavaPomArtifact.file
+            // println guavaPomArtifact.file
```

Comment:
❓ we could use the `guavaPomArtifact.file.name` instead, so that line can be uncommented and the assertions will be stable, wdyt? @eriwen?

Reply:
I'll do that. I don't think it hurts, and the reader should be smart enough to know that he can print the file if he can print its name